aiida_testing/mock_code/_fixtures.py

```diff
@@ -145,6 +162,12 @@ def _get_mock_code(
             assert isinstance(arg, collections.abc.Iterable) and not isinstance(arg, str), \
                 f"'ignore_files' and 'ignore_paths' arguments must be tuples or lists, found {type(arg)}"
 
+        if entry_point is None:
+            entry_point = label
+        if data_dir_abspath is None:
+            request.node.path.parent.joinpath("data").mkdir(exist_ok=True)
+            data_dir_abspath = request.node.path.parent / "data"
```

Comment:
@ltalirz I added the ability to just specify the label, e.g. `mock_code_factory('diff')`, in which case the entry point will be the same as the label, and the data directory is set as a `data` folder in the tests parent directory

Reply:
Thank you, these are good choices for defaults